_ml-commons-plugin/pretrained-models.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: default
+title: Pre-trained models
+parent: Model-serving framework
+nav_order: 120
+---
+
+# Pre-trained models
+
+The model-serving framework supports a variety of open-source pre-trained models that can assist with a variety of machine learning (ML) search and analytics use-cases. 
+
+## Uploading pre-trained models
+
+To use a pre-trained model in your OpenSearch cluster:
+
+1. Select the model and download the model artifact and config.json file. For a list of pre-trained models, see [supported pre-trained model](#supported-pre-trained-models).
+2. Upload the model using the [upload API]({{site.url}}{{site.baseurl}}/ml-commons-plugin/model-serving-framework#upload-model-to-opensearch). For more information on how to upload and use model, see [Model-serving framework]({{site.url}}{{site.baseurl}}/ml-commons-plugin/model-serving-framework).
+
+## Supported pre-trained models
+
+The model-serving framework supports the following models, categorized based on type. Although models with the same type will have similar use cases, each model has a different model size and performs differently depending on your cluster. For a comparison of the performances some pre-trained models, see the [sbert documentation](https://www.sbert.net/docs/pretrained_models.html#model-overview).
+
+
+### Sentence transformers
+
+Sentence transformer models map sentences and paragraphs across a dimensional dense vector space. The number of vectors depends on the model. Use these models for use cases such as clustering and semantic search.
+
+| **Model name** | **Vector dimensions** | **Torchscript artifact** | **ONNX artifact** |
```

Comment:
Is there a new standard that requires table headers to be bold? Should we be doing this across all docs?

Reply:
We don't have a standard one way or another. I always bold mine, so that the header labels are more obvious.

Since it's a formatting question, lets get a consensus and add it to the formatting guide.

Reply:
Agreed. They should generally always be bold.